app/src/main/java/org/oppia/app/fragment/FragmentComponent.kt

```diff
@@ -19,8 +19,8 @@ interface FragmentComponent {
     fun build(): FragmentComponent
   }
 
-  fun inject(audioFragment: AudioFragment)
   fun inject(explorationFragment: ExplorationFragment)
+  fun inject(audioFragment: AudioFragment)
```

Comment:
I think there are no changes in this file, in such cases make sure that you revert changes in such files, so as to make review process faster.

Reply:
This conversation has been marked as resolved but has not been solved yet.

Reply:
This would happen with several files due to merge, need to have verbal discussion on this .As if now iam reverting this file

Reply:
copied and develop code pasted the code as revert was not helping